dotcom-rendering/src/model/enhanceCollections.ts

```diff
@@ -7,9 +7,10 @@ import { groupCards } from './groupCards';
 
 const FORBIDDEN_CONTAINERS = [
 	'Palette styles new do not delete',
+	'Palette styles',
 	'culture-treat',
 	'newsletter treat',
-	'Palette styles',
+	'qatar treat',
```

Comment:
The US news treat does not need to be added here, right?

Reply:
Nope! Its already there just confusingly named `newsletter treat`. It already existed on another front but targetted a different container which is why i needed to define it a 2nd time in enhanceTreats